README.md

```diff
@@ -17,8 +17,21 @@ Psammead is a [GEL-compliant](https://www.bbc.co.uk/gel/articles/what-is-gel) Re
 Psammead packages are split into:
 
 - [Components](./packages/components) - GEL-compliant presentational React components, ready for use out of the box, regardless of data source.
+- [Containers](./packages/containers) - Functional components for optional use with presentational components of the same name.
 - [Utilities](./packages/utilities) - Commonly shared Psammead dependencies, and fundamentals to aid building additional GEL-compliant components.
 
+## Documentation index
+Please familiarise yourself with our:
+- [Code of conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md)
+- [Code Standards and Ways of Working](https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Workinnng.md)
```

Comment:
Typo. `https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Working.md`

Reply:
@mykeels done